layouts/partials/head.html

```diff
@@ -29,14 +30,12 @@
   crossorigin="anonymous"></script>
 {{ if .Site.Params.offlineSearch }}
 <script
-  src="https://unpkg.com/lunr@2.3.9/lunr.min.js"
-  integrity="sha384-203J0SNzyqHby3iU6hzvzltrWi/M41wOP5Gu+BiJMz5nwKykbkUx8Kp7iti0Lpli"
+  src="https://unpkg.com/lunr@2.3.8/lunr.min.js"
+  integrity="sha384-vRQ9bDyE0Wnu+lMfm57BlYLO0/XauFuKpVsZPs7KEDwYKktWi5+Kz3MP8++DFlRY"
```

Comment:
Hi again @jcabak. Why was lunr's version retrograded?

Could it be that, aside from the addition of `<meta name="description" ...>`, every other change in this file is because you were working from an old version of the file (including the removal of the `shortcodes.css` link below)?

For some context, see https://github.com/google/docsy/pull/540/files#r685339766.

/cc @LisaFC @nate-double-u